test/testdata/namer/fuzz_type_template_overwrite.rb

```diff
@@ -1,4 +1,5 @@
 # typed: false
+# disable-fast-path: true
```

Comment:
Since this file creates two static fields with the same name, the fast path now emits `Redefining constant` on lines 3 _and_ 4.